### ISSUE TITLE
New gateway/user does not see any sensor data (using container)

### ISSUE THREAD

**Issue by gvancuts** [01org/SmartHome-Demo#153]
After adding a new gateway and user (and mapping the two) in the cloud portal. The user dashboard does not display any sensor information although there are a number of them visible from that gateway. I observed that if I pick a pre-exisiting user (either dev or OTC) and make it point at the very same gateway, then I see all the sensor data.

This issue _may_ be confined to when running inside a container.

Here is how you can easily reproduce this problem:
1. Run the containers to have a complete system:
```
$ sudo docker run smarthome2cloud/smarthome-sensors
$ sudo docker run smarthome2cloud/smarthome-gateway
$ sudo docker run -p 3030:3000 -p 3031:4000  smarthome2cloud/smarthome-cloud
```

2. Log on to the Admin prtal (admin/admin) via: http://localhost:3031/
3. Add a new User
4. Add a new Gateway
5. Log on to the user portal via: http://localhost:3030/ , using the credentials you have put in via the Admin portal
=> There is no sensor data available (you can also verify that the gateway is correctly up and running by checking http://<ip-of-gateway-container>:8000/api/oic/d

6. Go back to the Admin portal
7. Modify an existing gateway to point at the very same gateway you started in a container (see above)
8. Log on to the User Portal using the user mapped to that gateway you've just updated (either dev or OTC)
=> you will see all the sensor data.

**Note**: it is not possible to restart the cloud container because as soon as you do that, the new user/gateway have disappeared (see #152)

**Comment by ichbinblau** [comment]
@gvancuts Yes, it is not possible for the time being.  A restart is required to load the data. I will find ways to improve it.

**Comment by gvancuts** [comment]
Let's update this to a low priority item and change it from a bug to a feature enhancement (see also why #152 was not a real bug but rather a buggy user ;-))
